### PR TITLE
Adding my folder, and automatic machine

### PR DESCRIPTION
**Pull request by phortela1n**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Sanitize linebreaks , spaces between lines

File: staff/pablo/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ $('#boton').prop('disabled', disable);
+});
+
+function showResults (value){
+
+  var array = calcula(value);
+  $("#showhere").html('Estás convirtiendo \n'+inputNumber+' '+ currency1+ ' a '+ currency2+': ' +'\n'+ value+' '+currency2 +' \n  esos '+inputNumber+' si fueran euros serían : '+'\n'+array).wrap('<pre />');
+  // 
+  
+
```
Comment (ghost) [inline staff/pablo/scripts/main.js]: So many line breaks

**Review by ghost — APPROVED** [review]
Also ok

**Comment by manuelbarzi** [comment]
👍

**Pull request merged**